sdks/java/core/src/main/java/org/apache/beam/sdk/coders/TimestampPrefixingWindowCoder.java

```diff
@@ -67,4 +74,20 @@ public T decode(InputStream inStream) throws CoderException, IOException {
   public void verifyDeterministic() throws NonDeterministicException {
     windowCoder.verifyDeterministic();
   }
+
+  @Override
+  public boolean consistentWithEquals() {
+    return windowCoder.consistentWithEquals();
+  }
+
+  @Override
+  public boolean isRegisterByteSizeObserverCheap(T value) {
+    return windowCoder.isRegisterByteSizeObserverCheap(value);
+  }
+
+  @Override
+  public void registerByteSizeObserver(T value, ElementByteSizeObserver observer) throws Exception {
+    InstantCoder.of().registerByteSizeObserver(new Instant(), observer);
```

Comment:
Did you mean to pass in the max timestamp of value instead of `new Instant()`?

Reply:
done.